dateparser/custom_lang_detect/__init__.py

```diff
@@ -0,0 +1,19 @@
+import importlib
+
+loaded_parser = None
+language_parser_module = None
+
+def language_parser(paser_module="fast_text"):
+  global loaded_parser
+  global language_parser_module
```

Comment:
No a good practice to use global variables, I will check it later and return to you with my suggestion.

Reply:
That will be great, I was thinking of creating a class should that be better